lib/database/ticket.php

```diff
@@ -392,8 +392,9 @@ function updateTicket($user, $ticketID, $ticketVal, $reason = null): bool
 
     addArticleComment("Server", ArticleType::AchievementTicket, $ticketID, $comment, $user);
 
-    if (!getAccountDetails($userReporter, $reporterData))
+    if (!getAccountDetails($userReporter, $reporterData)) {
         return true;
+    }
```

Comment:
The `braces` CS fixer rule would've demanded curly braces here.
The rule is disabled however because the indentations in php files in the public directory would be messed up because of it.
It will be enabled as soon as the files in the public directory have been refactored into controllers and views/components.